src/client/dfs/obj.c

```diff
@@ -616,8 +618,8 @@ dfs_obj_local2global(dfs_t *dfs, dfs_obj_t *obj, d_iov_t *glob)
 	oid_cp(&obj_glob->parent_oid, obj->parent_oid);
 	uuid_copy(obj_glob->coh_uuid, coh_uuid);
 	uuid_copy(obj_glob->cont_uuid, cont_uuid);
-	strncpy(obj_glob->name, obj->name, DFS_MAX_NAME + 1);
-	obj_glob->name[DFS_MAX_NAME] = 0;
+	memcpy(obj_glob->name, obj->name, DFS_MAX_NAME);
```

Comment:
.

Reply:
- [ ] Restore, add comments and pragma on `strncpy()` usage

Reply:
> * [x]  Restore, add comments and pragma on `strncpy()` usage

Fixed with commit f38fd44a0